### PR TITLE
Personalize CI success messages to IRC

### PR DESCRIPTION
**Pull request by cevich**
Rather than spamming the podman channel with impersonal success
messages referring to PR numbers, mention the author by nick name
and include the PR title and link.
    
Also avoid needless logging of all bot-script interactions with
IRC when there is no error detected.

### CODE REVIEW

**Comment by rhatdan** [comment]
Is there a translation table somewhere. Here I am rhatdan on irc I am dwalsh?

File: contrib/cirrus/git_authors_to_irc_nicks.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Comma separated mapping of author e-mail, to Freenode IRC nick.
+# When no match is found here, the username portion of the e-mail is used.
+# Sorting is done at runtime.
+# Comments (like this) and blank lines are ignored.
+
+rothberg@redhat.com,vrothberg
+santiago@redhat.com,edsantiago
+gscrivan@redhat.com,giuseppe
```
Comment (cevich) [inline contrib/cirrus/git_authors_to_irc_nicks.csv]: @rhatdan above is the translation table.  Note it's checking the commit author ***E-MAIL*** address, not the github login.  So in your case, no translation is needed :smile:

**Comment by cevich** [comment]
(rebase & force-push w/ added safety-net)

**Comment by cevich** [comment]
(something's not working)

**Comment by rhatdan** [comment]
LGTM
/approve

**Pull request merged**